.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: cpp 
-dist: trusty 
+dist: xenial 
```

Comment:
It seems that this requires updating the list of versioned dependencies (libboost1.55-dev, qt55base, etc). I would guess libboost-dev (i.e. 1.58) works. And probably Ubuntu's qt packages should be fine, instead of 3rd party.

Reply:
@maxGimeno Could you please have a look and make our Travis script work with "dist: xenial"? We will need that to switch to C++14.

Reply:
xenial already have a recent toolchain: https://docs.travis-ci.com/user/reference/xenial/#compilers-and-build-toolchain
> clang and llvm 7
> cmake 3.12.4
> gcc 5.4.0